.changelogs/7987.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Changed `afterAutofill` & `beforeAutofill` hooks' signatures",
+  "type": "changed",
+  "issue": 7987,
+  "breaking": true,
+  "framework": "none"
+}
```

Comment:
I think we need more changelog files

Reply:
Besides the `*Autofill` hook changes perhaps we could just say that this plugin is almost entirely different and link to the migration guide?

Reply:
Yup, something like that. Maybe mentioning that the dependencies change from `hot-formula-parser` to optional `hyperformula`

We won't have link to the migration guide before changelog so more context will be added in release notes and release blog post.

Reply:
How's this https://github.com/handsontable/handsontable/pull/7648/commits/6d7c3204fc9c5d3303fc06049d94cdac85c7b368

Reply:
👍